src/js/06-code.js

```diff
@@ -70,6 +70,29 @@ document.addEventListener('DOMContentLoaded', function () {
     document.body.removeChild(textarea)
   }
 
+  var casedLang = function (lang) {
```

Comment:
Will it fix https://github.com/neo4j-documentation/docs-ui/issues/9?

Reply:
I think this could be used for that. With your default suggestion of first-letter capitalisation as long as we don't have too many exceptions the list shouldn't get too long.